codec-socks/src/main/java/io/netty/handler/codec/socks/SocksCommonUtils.java

```diff
@@ -19,7 +19,7 @@
 import io.netty.util.CharsetUtil;
 import io.netty.util.internal.StringUtil;
 
-final class SocksCommonUtils {
+public final class SocksCommonUtils {
```

Comment:
Why public?

Reply:
oh, good catch. I can revert since methods have been moved to `ByteBufUtil`